### PR TITLE
#264 Added warning when vagga.yml is present but not vagga.yaml

### PR DESCRIPTION
**Pull request by anti-social**

### CODE REVIEW

File: src/config/config.rs [hunk 1]
```diff
@@ -41,25 +42,45 @@ pub fn config_validator<'a>() -> V::Structure<'a> {
...
+                    possibly it is a typo. \
+                    Correct configuration file name is vagga.yaml",
+                dir).ok();
+        }
+    }
+    
     let mut dir = work_dir.clone();
     loop {
         let fname = dir.join(".vagga/vagga.yaml");
         if fname.exists() {
```
Comment (tailhook) [inline src/config/config.rs]: Warning should be printed even if `vagga.yaml` is present too.

File: src/config/config.rs [hunk 2]
```diff
@@ -41,25 +42,45 @@ pub fn config_validator<'a>() -> V::Structure<'a> {
         command_validator()))
 }
 
-fn find_config_path(work_dir: &PathBuf) -> Option<(PathBuf, PathBuf)> {
+fn find_config_path(work_dir: &PathBuf, show_warnings: bool)
+    -> Option<(PathBuf, PathBuf)>
+{
+    fn maybe_print_warning(dir: &Path) {
```
Comment (tailhook) [inline src/config/config.rs]: Why this function is inside? I don't see clear reason keeping it there.

File: src/config/config.rs [hunk 3]
```diff
@@ -41,25 +42,45 @@ pub fn config_validator<'a>() -> V::Structure<'a> {
...
+        if dir.join("vagga.yml").exists() {
+            writeln!(&mut stderr(), "There is vagga.yml file in the {:?}, \
```
Comment (tailhook) [inline src/config/config.rs]: We usually to warnings with a `warn!` macro. The positive outcome of using `warn!` macro is that user can silence all warnings or some of them using `RUST_LOG`

File: src/config/config.rs (hunk 1 above)
Comment (anti-social) [inline src/config/config.rs]: > we can add a warning if there is vagga.yml but not vagga.yaml

I think this is reasonable to print warning if only incorrect file name is present.

File: src/config/config.rs (hunk 2 above)
Comment (anti-social) [inline src/config/config.rs]: Cause it is needed only to get rid of copy-pasting message.

File: src/config/config.rs (hunk 1 above)
Comment (tailhook) [inline src/config/config.rs]: So what to do if both are present? Isn't it confusing when you fix `vagga.yml` nothing changes?

Anyway, current fix looks fine.

**Pull request merged**